orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -8,6 +8,7 @@ base:
     - recipes.runner
     - fluent
     - metering
+    - ntp
```

Comment:
We should include a check if we're not on YCloud (Yarn) and skip all of these

Reply:
I considered doing that, but then I figured that when running on YCloud, either chrony won't be installed, or chronyd won't be running (because the containers shouldn't have CAP_SYS_TIME). Do you still think I should add an explicit check?

Reply:
In that case it's fine. I re-triggered the IT